TeamMemberApi/members/views.py

```diff
@@ -0,0 +1,83 @@
+from django.http import HttpResponse, JsonResponse
+from django.template import loader
+from rest_framework.renderers import JSONRenderer
+from rest_framework.parsers import JSONParser
+from members.models import Member
+from members.serializers import MemberSerializer
+from django.views.decorators.csrf import csrf_exempt, csrf_protect
+from django.views import generic
+from django.views.decorators.csrf import ensure_csrf_cookie
+
+"""
+    WARNING: Cross site requests not allowed. You can
+    add @csrf_exempt decorator above functions below
+    for testing purposes
+"""
+
+@csrf_protect
+def member_list(request):
```

Comment:
Django REST Framework not only provides serializers, but it also comes with powerful class-based views that can reduce your view code, check out http://www.django-rest-framework.org/tutorial/3-class-based-views/#rewriting-our-api-using-class-based-views